opt_einsum/backends/torch.py

```diff
@@ -41,14 +42,29 @@ def transpose(a, axes):
     return a.permute(*axes)
 
 
+@lru_cache(None)
+def get_einsum():
+    torch, _ = _get_torch_and_device()
+    if hasattr(torch, "_VF") and hasattr(torch._VF, "einsum"):
+        print("called")
+        return torch._VF.einsum
+    print("did not get called")
+
+    def einsum_no_opt(*args, **kwargs):
+        if hasattr(torch, "backends") and hasattr(torch.backends, "opt_einsum"):
```

Comment:
Minor comment - could this check be moved outside the function defn? I.e. if we can't find `_VF.einsum` or the opt_einsum flags, just return `torch.einsum` from `get_einsum`? Then it won't be checked every einsum call.

Reply:
done